clang/include/clang/Driver/Options.td

```diff
@@ -4370,7 +4370,9 @@ def fsycl : Flag<["-"], "fsycl">, Flags<[NoXarchOption, CoreOption]>, Group<sycl
   HelpText<"Enables SYCL kernels compilation for device">;
 def fno_sycl : Flag<["-"], "fno-sycl">, Flags<[NoXarchOption, CoreOption]>, Group<sycl_Group>,
   HelpText<"Disables SYCL kernels compilation for device">;
+// TODO: -fsycl-explicit-simd is deprecated. remove it when support is dropped.
 defm sycl_esimd: BoolFOption<"sycl-explicit-simd",
+  Group<clang_ignored_legacy_options_Group>,
   LangOpts<"SYCLExplicitSIMD">, DefaultFalse,
   PosFlag<SetTrue, [CC1Option], "Enable">, NegFlag<SetFalse, [], "Disable">,
   BothFlags<[NoArgumentUnused, CoreOption], "SYCL explicit SIMD extension.">>;
```

Comment:
Thanks for updating here - now that the deprecated message comes out and the option isn't hooked to anything, stating that the option does anything is misleading.  @pvchupin, how would be best to represent the information shown in the help?  Maybe something like `Deprecated - enabling explicit SIMD extensions is on by default`

Reply:
Fixed. Now it says:
```
  -fsycl-explicit-simd    Enable SYCL explicit SIMD extension. (deprecated)
  -fno-sycl-explicit-simd Disable SYCL explicit SIMD extension. (deprecated)
```